src/modules/fancyzones/lib/JsonHelpers.cpp

```diff
@@ -274,7 +274,7 @@ namespace JSONHelpers
         return modified;
     }
 
-    void FancyZonesData::CloneDeviceInfo(const std::wstring& source, const std::wstring& destination)
+    void FancyZonesData::CloneDeviceInfo(const std::wstring& source, const std::wstring& destination, bool newWorkArea)
```

Comment:
I wouldn't pass `newWorkArea` to `JsonHelpers` but rather just not call `CloneDeviceInfo` from `ZoneWindow` if it is new work area, because this argument basically cancels out entire function body.

Reply:
good point. will change that